src/cli.rs

```diff
@@ -1,21 +1,17 @@
-use std::{
-    net::{IpAddr, SocketAddr},
-    path::PathBuf,
-    time::Duration,
-};
+use std::{net::SocketAddr, path::PathBuf, time::Duration};
 
 use ::http::HeaderValue;
 use clap::{Args, Parser};
 use fqdn::FQDN;
-use hickory_resolver::config::CLOUDFLARE_IPS;
 use humantime::parse_duration;
 use ic_bn_lib::{
     http::{
-        self,
+        self, dns,
```

Comment:
just out of curiosity: why do import `dns` and use `dns::cli::Dns`, but then `vector::cli::Vector` and use `Vector`.